devices/sinope.js

```diff
@@ -972,8 +993,11 @@ module.exports = [
         description: 'Zigbee smart light switch',
         fromZigbee: [fz.on_off, fz.electrical_measurement, fzLocal.sinope],
         toZigbee: [tz.on_off, tzLocal.timer_seconds, tzLocal.led_intensity_on, tzLocal.led_intensity_off,
-            tzLocal.led_color_on, tzLocal.led_color_off],
+            tzLocal.led_color_on, tzLocal.led_color_off, tzLocal.keypad_lock],
         exposes: [e.switch(),
+            exposes.enum('action_report', ea.STATE, ['key_up_pressed', 'key_up_pressed2x', 'key_up_held',
+                'key_dn_pressed', 'key_dn_pressed2x', 'key_dn_held'])
```

Comment:
Does `dn` mean `down`? If yes, then please change it. Also rename `held` to `hold` and `pressed2x` to `pressed_2_times`

Reply:
I did a quick search in the device folder, and it appears the terms `single` `double` `hold` are more common to describe the clicks.  Are you ok with the following proposition instead:
```
            exposes.enum('action_report', ea.STATE, ['up_single', 'up_double', 'up_hold',
                'down_single', 'down_double', 'down_hold'])
```
defined in a similar way in inovelli.js and letv.js.